ipatests/test_xmlrpc/test_dns_plugin.py

```diff
@@ -2626,7 +2626,7 @@ def dns_setup(self, declarative_setup):
                 'result': {
                     'dn': idnzone1_dn,
                     'idnsname': [idnzone1_punycoded],
-                    'idnszoneactive': [u'TRUE'],
+                    'idnszoneactive': ['TRUE'],
```

Comment:
The quotes should be removed for consistency.

Reply:
Nope, this should not be removed. These are definitions for commands using `--raw` option and thus they will return strings as in LDAP entry. Same for all the others.

Reply:
Ah my bad, I missed this difference between raw and standard output. Thanks for the clarification